pkg/pquerier/frames.go

```diff
@@ -253,6 +264,20 @@ func (d *dataFrame) addMetricFromTemplate(metricName string, columnSize int, use
 	return nil
 }
 
+func (d *dataFrame) setDataAt(columnName string, index int, value interface{}) error {
+	colIndex, ok := d.columnByName[columnName]
+	if !ok {
+		return fmt.Errorf("no such column %v", columnName)
+	}
+	col := d.columns[colIndex]
+	err := col.SetDataAt(index, value)
```

Comment:
Can value be considered as empty? Value like NaN or empty string

Reply:
Currently i removed all places where i "set" NaN to a certain index so i only set "real" values.
Regarding string, because we only support raw data for string type there are no "empty strings"